test/unit/lib/kafka/producer.test.js

```diff
@@ -257,10 +257,11 @@ Test('Producer', producerTest => {
         test.end()
       } catch (e) {
         test.ok(e instanceof FSPIOPError)
-        test.ok(e.message === `The following Producers could not be disconnected: [{"topic":"${topicNameFailure}","error":"No producer found for topic ${topicNameFailure}"}]`)
+
+        test.ok(e.message === `The following Producers could not be disconnected: [{"topic":"${topicNameFailure}","error":"No producer found for topic ${topicNameFailure}: Sinon-provided No producer found for topic ${topicNameFailure}"}]`)
```

Comment:
It seems if you us an actual `Error` object on line `243` like so: 
```
getProducerStub.withArgs(topicNameFailure).throws(new Error(`No producer found for topic ${topicNameFailure}`))
```

this change will not be needed.